us.ihmc.cmap.CmapTools.yml

```diff
@@ -0,0 +1,50 @@
+app-id: us.ihmc.cmap.CmapTools
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: cmaptools
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk8
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-documents
+  - --persist=.cmaptools
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk8/install.sh
+
+  - name: cmaptools
+    buildsystem: simple
+    build-commands:
+      - chmod +x Linux64CmapTools_v6.04_09-24-19.bin
+      - ./Linux64CmapTools_v6.04_09-24-19.bin -i silent || true
+      - rm "${FLATPAK_DEST}/lib/cmaptools/Uninstall CmapTools"
+      - rm ${FLATPAK_DEST}/lib/cmaptools/CmapTools
+      - rm -r ${FLATPAK_DEST}/lib/cmaptools/UninstallerData
+      - install -Dm755 cmaptools.sh ${FLATPAK_DEST}/bin/cmaptools
+      - install -Dm644 us.ihmc.cmap.CmapTools.desktop ${FLATPAK_DEST}/share/applications/us.ihmc.cmap.CmapTools.desktop
+      - install -Dm644 us.ihmc.cmap.CmapTools.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/us.ihmc.cmap.CmapTools.png
+      - install -Dm644 us.ihmc.cmap.CmapTools.appdata.xml /app/share/appdata/us.ihmc.cmap.CmapTools.appdata.xml
+    sources:
+      - type: file
+        url: http://cmapdownload.ihmc.us/installs/CmapTools/Linux/Linux64CmapTools_v6.04_09-24-19.bin
```

Comment:
Is it redistributable?

Reply:
They sum up their license as free for commercial use.

Reply:
The license does not say.

Reply:
Then it should be assumed it's not, and reworked to use extra-data.

Reply:
Tried that. It seems I don't have Java during the extra-data step. :(

Reply:
You can always try contacting upstream for clarification.

Reply:
The [FAQ](https://cmap.ihmc.us/docs/cmaptools-faqs) states:

> **Can I get Access to the Source Code of CmapTools?**
No, we do not distribute the source code of CmapTools. The software is not Open Source, although CmapTools is _free to distribute_. Only the executable programs are available at the [CmapTools download](https://cmap.ihmc.us/cmaptools/cmaptools-download) page.